lib/util/fs/index.ts

```diff
@@ -43,6 +43,20 @@ export async function readLocalFile(
   }
 }
 
+export async function readLocalSymlink(
```

Comment:
please create a separate PR for those changes

Reply:
see #16673